lib/Dashboard/DeckWidgetUpcoming.php

```diff
@@ -44,7 +44,7 @@ public function __construct(IL10N $l10n) {
 	 * @inheritDoc
 	 */
 	public function getId(): string {
-		return 'deck';
+		return 'deckUpcoming';
```

Comment:
I'd prefer to keep this unchanged as otherwise the widget would disappear from user who have already setup as we use the id to store the dashboard layout and widget order.

Reply:
allright! I did.